.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
There are no comments on json files. @buzinas I don't use vscode. Are you ok with these changes?

Reply:
Yeah, I'm OK!